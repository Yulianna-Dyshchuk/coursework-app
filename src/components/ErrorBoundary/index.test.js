import React from "react";
import { shallow } from "enzyme";
import { ErrorBoundary } from "./index";

describe("ErrorBoundary component", () => {
  it("should render correctly without props", () => {
    const component = shallow(<ErrorBoundary />);

    expect(component).toMatchSnapshot();
  });

  it("should render correctly with children", () => {
    const component = shallow(
      <ErrorBoundary>
        <span>test child 1</span>
        <span>test child 2</span>
      </ErrorBoundary>
    );

    expect(component).toMatchSnapshot();
  });
});
