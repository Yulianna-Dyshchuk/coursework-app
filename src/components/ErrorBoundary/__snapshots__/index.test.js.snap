// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary component should render correctly with children 1`] = `
<Fragment>
  <span>
    test child 1
  </span>
  <span>
    test child 2
  </span>
</Fragment>
`;

exports[`ErrorBoundary component should render correctly without props 1`] = `<Fragment />`;
